rms/file_processing/clients.py

```diff
@@ -0,0 +1,19 @@
+from rms.file_processing.managers import StorageClient
+from azure.storage.blob.aio import BlobClient
+from fastapi import UploadFile
+
+
+class AzureBlobClient(StorageClient):
+
+    def __init__(self):
+        super().__init__()
```

Comment:
There is no need to override init just to call super

Reply:
fixed